Buildings/Experimental/DHC/Networks/Examples/Distribution1PipeExample.mo

```diff
@@ -0,0 +1,293 @@
+  connect(disAutoSize.port_bDisSup, supHeaWatAutoSize.ports[1]) annotation (
+      Line(points={{60,30},{68,30},{68,8},{-60,8},{-60,9}},  color={0,127,255}));
+  connect(disPlugFlow.port_bDisSup, supHeaWatPlugFlow.ports[1]) annotation (
+      Line(points={{60,-70},{72,-70},{72,-116},{-48,-116},{-48,-99}}, color={0,127,
+          255}));
+  connect(terAutoSize.port_bHeaWat,pumConAutoSize. port_a) annotation (Line(
+        points={{50,87.6667},{52,87.6667},{52,70}},                 color={0,127,
+          255}));
+  connect(pumConAutoSize.port_b, disAutoSize.ports_aCon) annotation (Line(
+        points={{52,50},{52,40}},                 color={0,127,255}));
+  connect(mFlowConAutosize.y, reaRep2.u)
+    annotation (Line(points={{99,60},{96,60}},   color={0,0,127}));
+  connect(reaRep2.y, pumConAutoSize.m_flow_in)
+    annotation (Line(points={{72,60},{64,60}}, color={0,0,127}));
+  connect(pumConPlugFlow.port_a, terPlugFlow.port_bHeaWat) annotation (Line(
+        points={{52,-30},{52,-14.3333},{50,-14.3333}}, color={0,127,255}));
+  connect(pumConPlugFlow.port_b, disPlugFlow.ports_aCon)
+    annotation (Line(points={{52,-50},{52,-60}}, color={0,127,255}));
+  connect(pumConPlugFlow.m_flow_in, reaRep3.y)
+    annotation (Line(points={{64,-40},{72,-40}}, color={0,0,127}));
+  connect(reaRep3.u, mFlowConPlugFlow.y)
+    annotation (Line(points={{96,-40},{99,-40}},   color={0,0,127}));
+  connect(pumAutoSize.port_a, supHeaWatAutoSize.ports[2]) annotation (Line(
+        points={{-20,30},{-40,30},{-40,11},{-60,11}},
+                                                    color={0,127,255}));
+  connect(pumPlugFlow.port_a, supHeaWatPlugFlow.ports[2]) annotation (Line(
+        points={{-20,-70},{-40,-70},{-40,-97},{-48,-97}},
+                                                        color={0,127,255}));
+  annotation (
+    Documentation(
+      info="<html>
+<p>
+Example model of two one-pipe distribution models that could be used i.e for building a reservoir network to connect several agents in series.  It showcases 
+<a href=\"modelica://Buildings.Experimental.DHC.Networks.Distribution1PipeAutoSize\">
+Buildings.Experimental.DHC.Networks.Distribution1PipeAutoSize</a> and 
+<a href=\"modelica://Buildings.Experimental.DHC.Networks.Distribution1PipePlugFlow\">
+Buildings.Experimental.DHC.Networks.Distribution1PipePlugFlow</a>. The distribution models create a vector of nLoa connection models <a href=\"modelica://Buildings.Experimental.DHC.Networks.Connections\">
+Buildings.Experimental.DHC.Networks.Connections</a> that are connected to a vector of nLoa agents made up by time series heating loads <a href=\"modelica://Buildings.Experimental.DHC.Loads.BaseClasses.Validation.BaseClasses.FanCoil2PipeHeatingValve\">
+Buildings.Experimental.DHC.Loads.BaseClasses.Validation.BaseClasses.FanCoil2PipeHeatingValve</a>. Each agent will draw water from the distribution pipe and release it to the same pipe.
```

Comment:
Avoid very long lines and put code into `<code>...</code>`

Reply:
Made lines shorter and added `<code>nLoa</code>` where necessary.